### PR TITLE
KubeSphere Certificate review

### PR DESCRIPTION
**Pull request by JiannaLiu**

### CODE REVIEW

File: content/en/blogs/kubesphere-certificates.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: 'Come on! Your Exclusive Certificate is Ready'
+tag: 'KubeSphere,Community'
+keyword: 'Community, Open Source, Contribution, KubeSphere'
+description: 'KubeSphere Community Steering Committee would like to express gratitude to KubeSphere Member, KubeSphere Ambassador, KubeSphere Talented Speaker, and KubeSphere Contributor in 2020'
+createTime: '2021-02-11'
+author: 'Feynman Zhou'
```
Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:7]: Add your name here.

File: content/en/blogs/kubesphere-certificates.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+
+
+It’s highly appreciated that the KubeSphere community has nearly 100 contributors  by February 2021. Voted and approved by the KubeSphere Community Technical Committee and the KubeSphere Steering Committee, 10 Ambassadors, 15 Talented Speakers, 22 outstanding contributors, and 6 active contributors with remarkable contributions were selected throughout the year.  
+
+
+
+## Vote of Thanks
+
+KubeSphere Community Steering Committee would like to express gratitude to KubeSphere Member, KubeSphere Ambassador, KubeSphere Talented Speaker, and KubeSphere Contributor, and extend the sincerest greetings to all the members who have participated in the open-source contribution of the KubeSphere community. Therefore, we prepare an exclusive gift for you.
```
Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:19]: KubeSphere Member, KubeSphere Ambassador, KubeSphere Talented Speaker, and KubeSphere Contributor
->
KubeSphere Members, KubeSphere Ambassadors, KubeSphere Talented Speakers, and KubeSphere Contributors

Comment (Sherlock113) [reply content/en/blogs/kubesphere-certificates.md:19]: who have participated in the open-source contribution of the KubeSphere community
->
who participated in the open-source contribution to the KubeSphere community

File: content/en/blogs/kubesphere-certificates.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+![合影](https://pek3b.qingstor.com/kubesphere-community/images/20210211222956.png)
+
+## KubeSphere Member
+
+KubeSphere Member is granted to members who have deeply participated in the open-source contribution of the KubeSphere community. Those members have independently completed one or more feature development, document writing and testing, also helped actively recommend KubeSphere in various communities and promote open source technology. Now we have invited all of them to join KubeSphere GitHub.
```
Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:25]: contribution of the KubeSphere community
->
contribution to the KubeSphere community

Comment (Sherlock113) [reply content/en/blogs/kubesphere-certificates.md:25]: Those members have independently completed one or more feature development, document writing and testing, also helped actively recommend KubeSphere in various communities and promote open source technology.
->
These members have independently developed or tested one or more features, written or translated documents, or helped to promote KubeSphere and open-source technologies in various communities.

Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:25]: KubeSphere Member is granted to
->
KubeSphere Members are granted to

File: content/en/blogs/kubesphere-certificates.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+|Jie Chen|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-chenjie.png) |
+|Ling Li|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-liling.png) |
+|Honglei Shen|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-shenhonglei.png) |
+|Turtle Chang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-turtlechang.png) |
+|Jian Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-zhagnjian.png) |
+|Kai Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-zhangkai.png) |
+
+## KubeSphere Talented Speaker
+
+KubeSphere Talented Speakers are awarded to outstanding lecturers who have conducted public technology presenation at KubeSphere Open-Source Live, KubeSphere Meetup 2020, and CNCF Webinar. They are excellent KubeSphere evangelists and splendid speakers.
```
Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:38]: conducted public technology presenation
->
conducted public technology presentations

File: content/en/blogs/kubesphere-certificates.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+|Xiaofei Yang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-yangxiaofei.png) |
+|Jiong Duan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-duanjiong.png) |
+|Wanjun Lei|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-leiwanjun.png) |
+|Michael Yuan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-michaelyuan.png) |
+|Wenhu Wang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-wangwenhu.png) |
+|Ke Zhou|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-zhouke.png) |
+
+## KubeSphere Ambassador
+
+KubeSphere Ambassador is awarded to technical evangelists who are helping grow the KubeSphere community. The following members have shared KubeSphere practical cases and technical articles in the KubeSphere community many times to help more users to get to know the application scenarios and best practices of KubeSphere.
```
Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:60]: KubeSphere Ambassador is awarded to technical evangelists who are helping grow the KubeSphere community.
->
KubeSphere Ambassadors are awarded to technical evangelists who are helping to grow the KubeSphere community.

File: content/en/blogs/kubesphere-certificates.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+|Yang Yang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-yangyang.png)|
+|Xingmin Lu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-luxingmin.png) |
+|Haitao Pan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-panhaitao.png) |
+|Will Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-willzhang.png) |
+|Xinglin Xue|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-xuexinglin.png) |
+|Haili Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/ambassador-zhanghaili.png) |
+
+## KubeSphere Contributor
+
+KubeSphere Contributor is for members who have participated in KubeSphere’s open-source project in 2020 or published high-quality technical blogs on the KubeSphere Chinese forum, as well as members who help to submit and fix bugs.。
```
Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:77]: ->
KubeSphere Contributors are for members who have participated in KubeSphere’s open-source projects in 2020 or published high-quality technical blogs on the KubeSphere Chinese forum, as well as members who help to submit and fix bugs.

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments.

File: content/zh/blogs/kubesphere-certificates.md [hunk 7]
```diff
@@ -5,7 +5,7 @@ keyword: '社区, 开源, 贡献, KubeSphere'
 description: 'KubeSphere 社区管理委员会向此次获得 2020 年度评选产生的 KubeSphere Member、 KubeSphere Ambassador、KubeSphere Talented Speaker、KubeSphere Contributor 成员致谢。'
 createTime: '2021-02-11'
 author: '周鹏飞'
-snapshot: 'https://pek3b.qingstor.com/kubesphere-community/images/20210211222956.png'
+snapshot: 'https://pek3b.qingstor.com/kubesphere-Twitter, Facebook, LinkedIncommunity/images/20210211222956.png'
```
Comment (Sherlock113) [inline content/zh/blogs/kubesphere-certificates.md:8]: Are you sure this link is accessible? There are spaces in the link.

File: content/zh/contribution/_index.md [hunk 8]
```diff
@@ -4,82 +4,82 @@ title: "参与贡献"
...
-    - name: 'Download'
+    - name: '下载'
       icon1: 'images/contribution/download.svg'
       icon2: 'images/contribution/37.png'
       children:
-        - content: 'Download KubeSphere'
+        - content: '下载 KubeSphere'
           link: '../../../zh/docs/quick-start/all-in-one-on-linux/'
-        - content: 'Quickstart'
+        - content: '快速开始'
```
Comment (Sherlock113) [inline content/zh/contribution/_index.md:19]: 快速开始
->
快速入门

File: content/zh/contribution/_index.md [hunk 9]
```diff
@@ -4,82 +4,82 @@ title: "参与贡献"
...
+      - name: '开发者社群'
         icon: 'images/contribution/developer-group.svg'
         children:
           - name: 'Owner /  Lead'
-            content: 'A lead is also a member of the project who is an experienced and active reviewer of the project.'
+            content: ' 作为项目的一员，Lead 具有丰富的经验，并且是位积极活跃的项目审阅者。'
 
           - name: 'Member'
-            content: 'A member has notable contributions to the project, helps us review PRs from other contributors.'
+            content: 'Member 对项目做出重大贡献，并帮助我们审阅其他贡献者的PR'
```
Comment (Sherlock113) [inline content/zh/contribution/_index.md:62]: 并帮助我们审阅其他贡献者的PR
->
并帮助我们审阅其他贡献者的 PR。

File: content/zh/contribution/_index.md [hunk 10]
```diff
@@ -4,82 +4,82 @@ title: "参与贡献"
...
 
           - name: 'Contributor'
-            content: 'Anyone who has at least one non-trivial PR merged into any project under KubeSphere organization becomes a contributor.'
+            content: '提交过一次重要PR且被合并到 KubeSphere 项目下的用户都将成为 Contributor'
```
Comment (Sherlock113) [inline content/zh/contribution/_index.md:65]: 提交过一次重要PR且被合并到 KubeSphere 项目下的用户都将成为 Contributor
->
提交过一次重要 PR 且被合并到 KubeSphere 项目下的用户都将成为 Contributor。

File: content/zh/contribution/_index.md [hunk 11]
```diff
@@ -4,82 +4,82 @@ title: "参与贡献"
...
-      - name: 'Steering Committee'
+      - name: '社区委员会'
         icon: 'images/contribution/steering.svg'
 
 section3:
   interestGroup:
-    title: 'Find Your Special Interest Group'
-    content: 'SIGs are designed to let you find everything you need in one place around a central topic. Find an interesting one and join the SIG.'
+    title: '加入您的特殊兴趣小组'
+    content: '如果您对某一个主题感兴趣，可以加入我们的 SIG。 在那里您可以找到与您感兴趣的主题所相关的一切内容。'
```
Comment (Sherlock113) [inline content/zh/contribution/_index.md:82]: Delete the space in front of the second sentence.

File: content/zh/contribution/_index.md [hunk 12]
```diff
@@ -269,22 +269,22 @@ sectionCalendar:
...
+        content: '您将会成为面向公众用户的 KubeSphere 社区代表。我们将为您提供资源，以帮助您成为一名成功的演讲者或作者。'
         link: 'request'
-        linkContent: 'Become a KubeSphere Ambassador →'
+        linkContent: '成为 KubeSphere Ambassador →'
 
-      - title: 'Be a Contributor'
+      - title: '成为 Contributor'
         icon: 'images/contribution/group-2.png'
-        content: 'KubeSphere is 100% open source and driven by the community. We appreciate contributions to our code and documentation! Significant contributions will earn you a contributor T-shirt.'
+        content: 'KubeSphere 100% 开放源代码，并由社区推动。 我们非常感谢对代码和文档做出贡献的用户！ 为了表达谢意，我们将为有突出贡献者赠送贡献者T恤。'
```
Comment (Sherlock113) [inline content/zh/contribution/_index.md:282]: KubeSphere 100% 开放源代码，并由社区推动。 我们非常感谢对代码和文档做出贡献的用户！ 为了表达谢意，我们将为有突出贡献者赠送贡献者T恤。
->
KubeSphere 100% 开放源代码，并由社区推动。我们非常感谢对代码和文档做出贡献的用户！为了表达谢意，我们将为有突出贡献者赠送贡献者 T 恤。

Add a space between English and Chinese characters, and do not add any space between two Chinese sentences.

File: content/zh/contribution/_index.md (hunk 8 above)
Comment (Sherlock113) [inline content/zh/contribution/_index.md:19]: 快速开始
->
快速入门

File: content/en/blogs/kubesphere-certificates.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+|Jie Chen|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-chenjie.png) |
+|Ling Li|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-liling.png) |
+|Honglei Shen|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-shenhonglei.png) |
+|Turtle Chang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-turtlechang.png) |
+|Jian Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-zhagnjian.png) |
+|Kai Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-zhangkai.png) |
+
+## KubeSphere Talented Speaker
+
+KubeSphere Talented Speakers are awarded to outstanding lecturers who have conducted public technology presenations at KubeSphere Open-Source Live, KubeSphere Meetup 2020, and CNCF Webinar. They are excellent KubeSphere evangelists and splendid speakers.
```
Comment (Sherlock113) [inline content/en/blogs/kubesphere-certificates.md:38]: presenations
->
presentations

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

File: content/zh/blogs/kubesphere-certificates.md (hunk 7 above)
Comment (FeynmanZhou) [reply content/zh/blogs/kubesphere-certificates.md:8]: @JiannaLiu It looks like the wrong URL is still not fixed, please use `https://pek3b.qingstor.com/kubesphere-community/images/20210211222956.png`.

**Pull request merged**